app/assets/scripts/views/project.js

```diff
@@ -72,6 +72,7 @@ var Project = React.createClass({
     const { lang } = this.props.meta;
     const basepath = '/' + lang;
     const ontime = ProjectCard.isOntime(data);
+    const ontimeLookup = {extended: 'Extended', delayed: 'Delayed', ontime: 'On Time'};
```

Comment:
Good example of why it should be in a utility function rather than hardcoded in multiple files.